marketpulse/base/mozillians.py

```diff
@@ -0,0 +1,85 @@
+import requests
+
+
+class BadStatusCode(Exception):
+    pass
+
+
+class ResourceDoesNotExist(Exception):
+    pass
+
+
+class MultipleResourcesReturned(Exception):
+    pass
+
+
+class MozilliansClient():
+    def __init__(self, api_base_url, api_key):
+        self.base_url = api_base_url
+        self.key = api_key
+        self.headers = {'X-API-KEY': self.key}
+
+    def _get(self, url, params=None):
+        """GET url, checks status code and return ApiResponse."""
+        response = requests.get(url, headers=self.headers, params=params)
+
+        if not response.status_code == 200:
+            raise BadStatusCode('{0} on: {1}'.format(response.status_code, url))
```

Comment:
I suggest we live it for now. It depends in the status code. For example ``500 error`` is returning the django error page. On the other hand, authentication errors return a valid json with an error message.